Atualiza o Guia para versão 6.1.4 do Rails (#514)
campuscode/rails-guides-pt-BR: pt-BR/active_record_migrations.md

`down`) de cada *migration*, você verá o texto `********** NO FILE **********`
próximo a cada arquivo de *migration* excluído que foi anteriormente executado
em um ambiente específico mas não se encontra mais no diretório `db/migrate/`.

> Porém, há uma advertência. As tarefas `rake` para instalar *migrations* de *engines* são idempotentes. As *migrations* presentes na aplicação principal devido a uma instalação anterior são ignoradas e as que faltam são copiadas com um novo carimbo de data/hora (*timestamp*) inicial. Se você excluiu as *migrations* de *engines* antigas e executou a tarefa de instalação novamente, você obteria novos arquivos com novos carimbos de data/hora e `db: migrate` tentaria executá-los novamente.

Author: ```suggestion
Porém, há uma advertência. As tarefas `rake` para instalar *migrations* de *engines* são idempotentes. As *migrations* presentes na aplicação principal vindas de uma instalação anterior são ignoradas e as que faltam são copiadas com um novo carimbo de data/hora (*timestamp*). Se você excluiu as *migrations* de *engines* antigas e executou a tarefa de instalação novamente, você obteria novos arquivos com novos carimbos de data/hora e e o comando `db: migrate` tentaria executá-las novamente.
```

> Porém, há uma advertência. As tarefas `rake` para instalar *migrations* de *engines* são idempotentes. As *migrations* presentes na aplicação principal vindas de uma instalação anterior são ignoradas e as que faltam são copiadas com um novo carimbo de data/hora (*timestamp*). Se você excluiu as *migrations* de *engines* antigas e executou a tarefa de instalação novamente, você obteria novos arquivos com novos carimbos de data/hora e e o comando `db: migrate` tentaria executá-las novamente.

Reviewer: ```suggestion
Porém, há uma advertência. As tarefas `rake` para instalar *migrations* de *engines* são idempotentes. As *migrations* presentes na aplicação principal vindas de uma instalação anterior são ignoradas e as que faltam são copiadas com um novo carimbo de data/hora (*timestamp*). Se você excluiu as *migrations* de *engines* antigas e executou a tarefa de instalação novamente, você obteria novos arquivos com novos carimbos de data/hora e o comando `db: migrate` tentaria executá-las novamente.
```

Author: Mudei no 5d766b1597fcc6893b834c860f44be5dcd595512

> Portanto, geralmente você deseja preservar as *migrations* provenientes de *engines*. Eles têm um comentário especial como este:

Author: ```suggestion
Portanto, geralmente você deseja preservar as *migrations* provenientes de *engines*. Elas têm um comentário especial como este:
```
